InventoryReservationCli/Model/SalableQuantityInconsistency/AddExpectedReservations.php

```diff
@@ -102,4 +102,20 @@ private function calculateReservationQty(array $data): float
 
         return (float)$qty;
     }
+
+    /**
+     * Return reservation qty amount
+     *
+     * @param array $data
+     * @return float
+     */
+    private function calculateReservationQty(array $data): float
+    {
+        $qty = $data['qty_ordered'];
+        $qty -= $data['qty_canceled'];
+        $qty -= $data['qty_refunded'];
+        $qty -= $data['is_virtual'] ? $data['qty_invoiced'] : $data['qty_shipped'];
+
+        return (float)$qty;
+    }
```

Comment:
This code is not part of ACP2E-294 or MC-42066, but of MC-41222 that has been backported in MDVA-37291 (see https://github.com/magento-sparta/inventory/pull/27/files#diff-55972798e3b836abd2d2297cadd76ae53ead9e2a0a319647b0363f8f373ccdc9R89). As a result, if you check in https://github.com/magento-sparta/inventory/pull/158/files#diff-55972798e3b836abd2d2297cadd76ae53ead9e2a0a319647b0363f8f373ccdc9R89 you can see that in your PR you have 2 methods with the same name "calculateReservationQty"

Reply:
@robertorubino Fixed. Thanks for catching this